### PR TITLE
Toolbar buttons

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

A mixin to solve the button with tooltip alignment issue.

# Changes

- new mixin `toolbar`.
- replace custom css in modals with the `toolbar` mixin.

# Screenshots

Disburse maturity. The jumping is another issue because of the tooltip position.
Screen Recording 2024-01-31 at 12 02 05

Participation button
image

### CODE REVIEW

File: src/lib/styles/global/modal.scss [hunk 1]
```diff
@@ -14,21 +14,8 @@ div.modal {
   }
 
   .toolbar {
-    display: flex;
```
Comment (dskloetd) [inline src/lib/styles/global/modal.scss:17]: Does this mean that existing usages of `.toolbar` will break unless they also add `.toolbar-buttons`?

Comment (mstrasinskis) [reply src/lib/styles/global/modal.scss:17]: Yes, the plan was to add the `.toolbar-buttons` class together with the `gix-upgrade` pr.

Comment (dskloetd) [reply src/lib/styles/global/modal.scss:17]: What do you think about moving the `toolbar` class outside of `div.modal` so we can reuse it without having to add multiple classes each time?
Or would that not work?

Comment (mstrasinskis) [reply src/lib/styles/global/modal.scss:17]: Not sure, 
1. it has an extra padding inside the modal;
2. it's a bit confusing that there will be a global class `.toolbar` that is not related directly to the `Toolbar` component.

What do you think about renaming `.toolbar-buttons` in this pr to `.full-width-mobile-buttons`?

Comment (dskloetd) [reply src/lib/styles/global/modal.scss:17]: I'm concerned that using `.toolbar` inside `div.modal` used to work and now requires an extra class.

How about also adding `.toolbar` style that works inside `BottomSheet`?

I'm also still confused how this even works. Why does the `div.modal` selector here work without the need for `:global`?

Comment (dskloetd) [reply src/lib/styles/global/modal.scss:17]: Then why did it have `:global(button) {` on line 22?

Comment (mstrasinskis) [reply src/lib/styles/global/modal.scss:17]: Good question, but it doesn't work w/o `:global`

Comment (dskloetd) [reply src/lib/styles/global/modal.scss:17]: Thanks for checking.

Are you still looking into this:
> I'm concerned that using .toolbar inside div.modal used to work and now requires an extra class.
> How about also adding .toolbar style that works inside BottomSheet?

Or did you miss that?

Comment (mstrasinskis) [reply src/lib/styles/global/modal.scss:17]: A [mixin](https://github.com/dfinity/gix-components/pull/375/files#diff-bbba86cbcfd780d497c1fc89e609c3599beee6594f8bdb01bcc8195210e97565R20) was added.
- no changes needed for modals
- can be used to fix the "Participate" button issue -- https://github.com/dfinity/nns-dapp/pull/4401

Comment (mstrasinskis) [reply src/lib/styles/global/modal.scss:17]: @dskloetd, pr was updated:
- `*` selectors to make it most flexible
- keep only `base` version of the mixin

File: src/lib/styles/mixins/_toolbar.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
+@use "./media";
+
+// A mixin to align buttons (also wrapped).
+// Uses all available space to aligns the buttons on mobile. Don't resize/alight buttons on desktop.
```
Comment (dskloetd) [inline src/lib/styles/mixins/_toolbar.scss:4]: typo: alight

**Review by dskloetd — APPROVED** [review]
Thanks!

File: src/lib/styles/mixins/_toolbar.scss (hunk 2 above)
Comment (mstrasinskis) [reply src/lib/styles/mixins/_toolbar.scss:4]: Thanks.

**Pull request merged**